internal/integration/registry_test.go

```diff
@@ -69,7 +69,9 @@ func testInstallerImage(ctx context.Context, t *testing.T, registry name.Registr
 	layers, err := img.Layers()
 	require.NoError(t, err)
 
-	assert.Len(t, layers, 2)
+	if talosVersion != "v1.3.7" {
+		assert.Len(t, layers, 2)
```

Comment:
let's also add some context message for the error, `3 != 2` never really helps :sweat_smile:

Reply:
which error? I just skip the assertion for v1.3.7, which doesn't have 2 layers

Reply:
i meant for the `assert` if it fails just says `x !=2` without much context why it failed

Reply:
I'm not sure how it can be made more helpful? if you go back to the line you know why it failed, but if I make custom message, iirc it will skip the actual `3 != 2`, so you are left to wonder what was the length

Reply:
oh, I'm wrong, it still prints the assertion

Reply:
thanks, updated